app/forms/flood_risk_engine/steps/local_authority_form.rb

```diff
@@ -0,0 +1,47 @@
+module FloodRiskEngine
+  module Steps
+    class LocalAuthorityForm < FloodRiskEngine::Steps::BaseForm
+
+      def self.factory(enrollment)
+        organisation = enrollment.organisation || Organisation.new
+        new(organisation, enrollment)
+      end
+
+      def self.locale_key
+        "flood_risk_engine.enrollments.steps.local_authority"
+      end
+
+      def self.name_max_length
+        200
+      end
+
+      validation :name_present? do
+        validates :name, presence: {
+          message: I18n.t("#{LocalAuthorityForm.locale_key}.errors.name.blank")
+        }
+      end
```

Comment:
as above ... this allows the validation result to be used elsewhere allowing the more readable

```validation :validate_name_format, if: :name_present? do```